src/BizHawk.Client.EmuHawk/CustomControls/HexTextBox.cs

```diff
@@ -15,7 +15,7 @@ public interface INumberBox
 		void SetFromRawInt(int? rawInt);
 	}
 
-	public class HexTextBox : TextBox, INumberBox
+	public class HexTextBox : ClipboardEventTextBox, INumberBox

```

Comment:
Can the `INumberBox` inheritance be moved to the base class?

Reply:
Should it be? The relevant implementations (`HexTextBox` and `WatchValueBox`) are all different, and while `ClipboardEventTextBox` might never actually be used for anything else, its functionality is not tied to numeric inputs.